ownCloud/Client/PhotoSelectionViewController.swift

```diff
@@ -241,11 +241,11 @@ class PhotoSelectionViewController: UICollectionViewController, Themeable {
 		// Add a badge to the cell if the PHAsset represents a Live Photo.
 		if asset.mediaSubtypes.contains(.photoLive) {
 			cell.mediaBadgeImage = PHLivePhotoView.livePhotoBadgeImage(options: .overContent)
-		}
-
-		if asset.mediaType == .video {
+		} else if asset.mediaType == .video {
 			cell.videoDurationLabel.text = durationFormatter.string(from: asset.duration)
 			cell.mediaBadgeImage = UIImage(named: "camera-badge")?.withRenderingMode(.alwaysTemplate).tinted(with: UIColor.white)
+		} else if PHAssetResource.assetResources(for: asset).filter({$0.type == .alternatePhoto}).count > 0 {
+			cell.mediaBadgeImage = UIImage(named: "raw-badge")?.withRenderingMode(.alwaysTemplate).tinted(with: UIColor.white)
```

Comment:
In this and other calls following the pattern
```swift
UIImage(named: imageName)?.withRenderingMode(.alwaysTemplate).tinted(with: UIColor.white)
```
the icon is re-computed anew for every cell, and every cell then holds onto its own `UIImage` instance. Computing these icons only once - and sharing them between cells would be a quick performance and memory savings win.